master/internal/resourcemanagers/resource_managers.go

```diff
@@ -3,13 +3,18 @@ package resourcemanagers
 import (
 	"time"
 
+	"github.com/determined-ai/determined/master/internal/config"
 	"github.com/determined-ai/determined/master/internal/job"
 	"github.com/determined-ai/determined/master/internal/sproto"
 	"github.com/determined-ai/determined/master/pkg/actor"
 )
 
 const (
-	actionCoolDown = 500 * time.Millisecond
+	// DefaultSchedulingPriority is the default resource manager priority.
+	DefaultSchedulingPriority = 42
+
+	actionCoolDown          = 500 * time.Millisecond
+	defaultResourcePoolName = "default"
```

Comment:
Wait - I don't see where DefaultSchedulingPriority or defaultResourcePoolName came from before. Missing something?

Reply:
`scheduler_config.go` moved into the config package; so I copied some consts back to `resource_managers.go` 

Here's the original block on master:
https://github.com/determined-ai/determined/blob/master/master/internal/resourcemanagers/scheduler_config.go#L15

I thought this was conceptually cleaner than having the exported from config, but maybe it's better overall not to maintain duplicate constants. I'll make this change inline with the fix Ilia suggested above.